content/api/metrics.apib

```diff
@@ -170,6 +170,7 @@ Filter keys must be unique within a grouping, but the same filter key can appear
                subject_campaigns
                templates
                sending_domains
+               mailbox_provider
```

Comment:
Should be plural, and please merge with the master branch again as I just pushed out the mailbox_provider_regions.

Reply:
good catch